Modules/ThirdParty/GDCM/src/gdcm/CMakeLists.txt

```diff
@@ -1,14 +1,7 @@
-cmake_minimum_required(VERSION 3.9.2 FATAL_ERROR)
+set(GDCM_MAX_VALIDATED_CMAKE_VERSION "3.13.4")
```

Comment:
Is this the only file we bundle in ITK? Upstream PR has more changes.

Reply:
In our bundled versions, the best I can tell that code is not reachable.